src/math/BesselY.cpp

```diff
@@ -0,0 +1,1432 @@
+        //         in DASYJY for orders FNU and FNU+1 to start the recursion.
+        //         NULIM is 70 or 100 depending on whether N=1 or N .GE. 2.  An
+        //         overflow test is made on the leading term of the asymptotic
+        //         expansion before any extensive computation is done.
+        //
+        //         The maximum number of significant digits obtainable
+        //         is the smaller of 14 and the number of digits carried in
+        //         double precision arithmetic.
+        //
+        //     Description of Arguments
+        //
+        //         Input
+        //           X      - X .GT. 0.0D0
+        //           FNU    - order of the initial Y function, FNU .GE. 0.0D0
+        //           N      - number of members in the sequence, N .GE. 1
+        //
+        //         Output
+        //           Y      - a vector whose first N components contain values
+        //                    for the sequence Y(I)=Y/sub(FNU+I-1)/(X), I=1,N.
+        //
+        //     Error Conditions
+        //         Improper input arguments - a fatal error
+        //         Overflow - a fatal error
+        //
+        // ***REFERENCES  F. W. J. Olver, Tables of Bessel Functions of Moderate
+        //                 or Large Orders, NPL Mathematical Tables 6, Her
+        //                 Majesty's Stationery Office, London, 1962.
+        //               N. M. Temme, On the numerical evaluation of the modified
+        //                 Bessel function of the third kind, Journal of
+        //                 Computational Physics 19, (1975), pp. 324-337.
+        //               N. M. Temme, On the numerical evaluation of the ordinary
+        //                 Bessel function of the second kind, Journal of
+        //                 Computational Physics 21, (1976), pp. 343-350.
+        // ***ROUTINES CALLED  D1MACH, DASYJY, DBESY0, DBESY1, DBSYNU, DYAIRY,
+        //                    I1MACH, XERMSG
+        // ***REVISION HISTORY  (YYMMDD)
+        //    800501  DATE WRITTEN
+        //    890531  Changed all specific intrinsics to generic.  (WRB)
+        //    890911  Removed unnecessary intrinsics.  (WRB)
+        //    890911  REVISION DATE from Version 3.2
+        //    891214  Prologue converted to Version 4.0 format.  (BAB)
+        //    900315  CALLs to XERROR changed to CALLs to XERMSG.  (THJ)
+        //    920501  Reformatted the REFERENCES section.  (WRB)
+        //    170203  Converted to C++, and modified to only cover n==1 option. (MJ)
+        // ***END PROLOGUE  DBESY
+
+        const double xlim = std::numeric_limits<double>::min() * 1.e3;
+        const double elim = -std::log(xlim);
+        const int nulim[2] = { 70, 100 };
+
+        assert(fnu >= 0.);
+        assert(x > 0.);
+
+        if (x < xlim)
+            throw std::runtime_error("DBESY OVERFLOW, FNU OR N TOO LARGE OR X TOO SMALL");
```

Comment:
Same question about this runtime error as above (text vs. test mismatch).

Reply:
Ditto.  There are 3 other cases that emit this error message, which can be because nu is large.